# -*- coding: utf-8 -*-
"""Копия блокнота "Лабораторная работа 3 Пи-311 Павленко А.Х."

Automatically generated by Colaboratory.

Original file is located at
    https://colab.research.google.com/drive/1nZxMzFAWScJrysBlvYFkrjEiJl6gMG7H

задание 1
"""

n=input()
if n=="01":
  print ("January")

"""Задание 2"""

n=int(input())
if n % 2==0:
  print("Четное")
else:
   print("Нечетное")

"""Задание 3"""

N=int(input())
print("N=",N)
if N>40:
  print ("stonks")
else: print("not stonks")

"""Задание 4"""

def is_year_leap():
  year = int(input())
  if year % 4 == 0 or (year % 100 != 0 and year % 400 == 0):
    return (True)
  else: return False
is_year_leap()

"""Задание 5"""

def is_prime():
  n=int(input())
  d=n+1
  if n%d!=0:
    
    return True
  else: return False
is_prime()

"""Задание 6

"""

A=float(input())
B=float(input())
A1=A
B1=B
f=3.6
d=((138/2)**1.3)
d2=d*-1
g=((69/28**5.1)*4)
g2=g*-1
if A/B==f:
  A=B1
  B=A1
elif d2<=B<=g2:
     A=B1
     B=A1
print("A=",A)
print("B=",B)

"""Задание 7

"""

a=int(input())
b=int(input())
c=int(input())
d=int(input())
e=int(input())
a1=0
a2=a1
b1=0
b2=b1
c1=0
c2=c1
if a%2==0:
  a2=a2+1
a1=a2
if a<0:
  b2=b2+1
b1=b2
if -256<=a<=1024:
  c2=c2+1
c1=c2 


if b%2==0:
  a2=a2+1
a1=a2
if b<0:
  b2=b2+1
b1=b2
if -256<=b<=1024:
  c2=c2+1
c1=c2 

if c%2==0:
  a2=a2+1
a1=a2
if c<0:
  b2=b2+1
b1=b2
if -256<=c<=1024:
  c2=c2+1
c1=c2 


if d%2==0:
  a2=a2+1
a1=a2
if d<0:
  b2=b2+1
b1=b2
if -256<=d<=1024:
  c2=c2+1
c1=c2 


if e%2==0:
  a2=a2+1
a1=a2
if e<0:
  b2=b2+1
b1=b2
if -256<=e<=1024:
  c2=c2+1
c1=c2 
print("--------------------------")
print("четных=",a1)
print("отрицательных=",b1)
print("В промежутке[-256; 1024]=",c1)

"""Задание повышенной сложности """

В файле лабораторной сломалось изображение формулы. Выполнить задание не могу.